Makefile

```
@@ -43,11 +43,12 @@ docs-serve: ## Preview the documentation
 .PHONY: compile
 .compile:
```

This breaks `make compile` for me.

Shouldn't it be?
```suggestion
compile:
```

---

yes correct, nevertheless I will create a followup PR and the `Makefile` will be managed by modulesync

---

I think @Kidswiss should run `make test` instead?

---

@ccremer ah isn't `make compile` intended to be called directly?